testing/integration/proxy_url_test.go

```diff
@@ -302,6 +306,7 @@ func TestProxyURL_NoEnrollProxyAndProxyInThePolicy(t *testing.T) {
 
 func TestProxyURL_RemoveProxyFromThePolicy(t *testing.T) {
 	_ = define.Require(t, define.Requirements{
+		Group: "default",
```

Comment:
Might be worth making the default group a constant in the testing package, mostly to prevent accidentally creating a different test group with a typo.

Reply:
Added.